### PR TITLE
Node drain

### PR DESCRIPTION
**Pull request by Arau**
This PR has the intent of reviewing the documentation rather than merging to master. In fact, the latest version of the docker image is not yet available hence the documentation would be too forward.

### CODE REVIEW

File: _docs/operations/maintenance.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+title: StorageOS Docs - Maintenance
+anchor: operations
+module: operations/maintenance
+---
+
+# Maintenance
+
+## Node drain
+
+It is recommended to drain a node before any maintenance operation.
```
Comment (croomes) [inline _docs/operations/maintenance.md]: Should have a brief description of what node drain does?  e.g. "Node drain will move any active volumes from the node and re-schedule them on other nodes in a cluster.  This operation will take longer for large volumes."

File: _docs/operations/maintenance.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+#### Hosting primary volumes with replication *not enabled*:
+
+- A replica will be created in a different available node 
+- The new replica will be promoted to primary
+- The old primary volume will be removed
+
+
+#### If there are no available nodes free
+
+There is a variety of reasons for which a volume can not be hosted by a node. For instance, because there are rules in place that define restrictions of location, or because the amount of replicas +1 match the amount of nodes in the cluster.
```
Comment (croomes) [inline _docs/operations/maintenance.md]: "There is a variety" -> "There are a variety"

Comment (Arau) [reply _docs/operations/maintenance.md]: Thank you!
Fix commited: https://github.com/storageos/storageos.github.io/pull/145/commits/8e27e6f1a50283b43209c1ebd1ed6496014a0358

File: _docs/operations/maintenance.md (hunk 1 above)
Comment (Arau) [reply _docs/operations/maintenance.md]: Added clarification: https://github.com/storageos/storageos.github.io/pull/145/commits/784fbe8de26d2f9c6585e52264e750a2a38be230

File: _docs/operations/maintenance.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+#### Hosting primary volumes with replication *not enabled*:
+
+- A replica will be created in a different available node 
+- The new replica will be promoted to primary
+- The old primary volume will be removed
+
+
+#### If there are no available nodes free
+
+There are a variety of reasons for which a volume can not be hosted by a node. For instance, because there are rules in place that define restrictions of location, or because the amount of replicas +1 match the amount of nodes in the cluster.
```
Comment (oicheryl) [inline _docs/operations/maintenance.md]: "for which" => "why"
"can not" => "cannot"

Comment (Arau) [reply _docs/operations/maintenance.md]: Thank you!!!

Done https://github.com/storageos/storageos.github.io/pull/145/commits/becce5d7ce8ed3bf35ac6bfd8c6ff40acf5d22a7

**Pull request merged**